cmd/uastats/main.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"text/tabwriter"
+
+	"code.avct.io/uasurfer"
+)
+
+func main() {
+	var count int
+	ua := &uasurfer.UserAgent{}
+	stats := stats{
+		BrowserNames: make(map[uasurfer.BrowserName]int),
+		OSNames:      make(map[uasurfer.OSName]int),
+		DeviceTypes:  make(map[uasurfer.DeviceType]int),
+	}
+
+	scanner := bufio.NewScanner(os.Stdin)
+	for scanner.Scan() {
+		count++
+		ua.Reset()
+		uasurfer.ParseUserAgent(scanner.Text(), ua)
+		stats.BrowserNames[ua.Browser.Name]++
+		stats.OSNames[ua.OS.Name]++
+		stats.DeviceTypes[ua.DeviceType]++
+	}
+	if err := scanner.Err(); err != nil {
+		fmt.Fprintln(os.Stderr, "reading standard input:", err)
+	}
+	fmt.Printf("Read %d useragents\n", count)
+	fmt.Println()
```

Comment:
Is this just used to print a blank line?

Reply:
Yep

Reply:
Wasn't sure if you'd forgot to put some content in there ;)